pkg/scanners/trivy/trivy.go

```diff
@@ -188,7 +187,11 @@ func main() {
 
 		log.Info("scanning image", "imageRef", imageRef)
 
-		dockerImage, cleanup, err := fanalImage.NewDockerImage(ctx, imageRef, scanConfig.dockerOptions)
+		dockerImage, cleanup, err := fanalImage.NewContainerImage(ctx, imageRef, scanConfig.dockerOptions,
+			fanalImage.DisablePodman(),
+			fanalImage.DisableDockerd(),
```

Comment:
is this going to prevent dockershim scans?

Reply:
Yes. Probably the best move is to disable dockerd unless the runtime is dockershim